basics/data-import/pinot-stream-ingestion/apache-pulsar.md

```diff
@@ -63,12 +63,22 @@ You can change the following Pulsar specifc configurations for your tables
 
 ### Authentication
 
-The Pinot-Pulsar connector supports authentication using the security tokens. You can generate the token by following the [official Pulsar documentaton](https://pulsar.apache.org/docs/en/security-token-client/). Once generated, you can add the following property to `streamConfigs` to add auth token for each request
+The Pinot-Pulsar connector supports authentication using the security tokens. You can generate the token by following the [official Pulsar documentation](https://pulsar.apache.org/docs/en/security-token-client/). Once generated, you can add the following property to `streamConfigs` to add auth token for each request
```

Comment:
```suggestion
The Pinot-Pulsar connector supports authentication using security tokens. To generate a token, follow the instructions in [Pulsar documentation](https://pulsar.apache.org/docs). Once generated, add the following property to `streamConfigs` to add an authentication token for each request:
```

Reply:
appears the link is no longer working, added link to top-level docs site

Reply:
Sure. I was just correcting a typo there. I have found the updated link to the JWT token auth page.